src/modules/user/components/user-contact-popup/user-contact-popup.tsx

```diff
@@ -8,33 +8,66 @@ import type { UserContactPopupProps } from './user-contact-popup.interface';
 import styles from './user-contact-popup.module.scss';
 
 export const UserContactPopup: FC<UserContactPopupProps> = ({ ...props }) => {
+  const handlePhoneClick = () => {
```

Comment:
I think in this specific case you may write this right inside onClick. I don't really like that we have to do a lot of checks before making some action. And in this particular case it won't affect performance.